azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -60,6 +105,10 @@ Currently, automatic extension upgrades are only supported in the Windows Admin
 
 ### Enable automatic upgrade via the Azure portal
```

Comment:
Since this is a H3 heading, the subheadings for The Azure portal and Azure CLI should be H4 (####).